package.json

```diff
@@ -5,9 +5,9 @@
   "description": "Ferdi server to replace the default Franz server.",
   "main": "index.js",
   "engines": {
-    "node": "^14.17",
-    "npm": "^6.14",
-    "node-gyp": "^8.0"
+    "node": "^16.14.2",
```

Comment:
I prefer not to have open-ended dependencies. Please change to locking for specific versions.